src/coreclr/jit/fginline.cpp

```diff
@@ -405,55 +395,23 @@ class SubstitutePlaceholdersAndDevirtualizeWalker : public GenTreeVisitor<Substi
         // candidates.
         //
         // Do the deferred work now.
-        if (retClsHnd != NO_CLASS_HANDLE)
+        if ((*use)->IsCall() && (*use)->AsCall()->HasMultiRegRetVal())
```

Comment:
Does this not cause diffs on platforms with multi-reg `LONG` returns?

Reply:
The 32 bit builds certainly look unhappy.